releases.json

```diff
@@ -887,6 +887,7 @@
       "mpglib"
     ],
     "versions": [
+      "3.100-6",
```

Comment:
You'll need to reorder this one to the front (and maybe co-authored-by it?).

Reply:
Not sure what you mean.

Reply:
After the version bump come 10 commits introducing further changes. These should all come *before* the bump?

![imagen](https://user-images.githubusercontent.com/13498015/204169846-49409412-bf4c-49bd-ad53-c945ba384ac7.png)

Reply:
Fixed.